### DISCUSSION TITLE
In outpost deathmatch, job specific spawnpoints don't work

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15401] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

we already have "any job" spawnpoints, so why force the players to spawn at random ones regardless of the selected job?

### Reproduction steps

1. add a spawnpoint
2. set it to a job
3. watch in dismay as outpost deathmatch mode spawns a different job at that spawnpoint

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in our private repo now, fix coming up in the next unstable build and the next release.
